internal/tiltfile/tiltfile_state.go

```diff
@@ -1004,28 +1004,33 @@ func (s *tiltfileState) extractEntities(dest *k8sResource, imageRef container.Re
 }
 
 // If the user requested only a subset of manifests, filter those manifests out.
-func match(manifests []model.Manifest, matching map[string]bool) ([]model.Manifest, error) {
-	if len(matching) == 0 {
+func match(manifests []model.Manifest, requestedManifests []model.ManifestName) ([]model.Manifest, error) {
```

Comment:
...a nit and i know you didn't write this line of code, but the comment above makes it seem like we're EXcluding the requested manifests. maybe: "...return only those manifests"? but also 🤷‍♀